### ISSUE TITLE
create-remix always uses the latest version (2.2.0) instead of the specified version

### ISSUE THREAD

**Issue by andrelandgraf** [remix-run/remix#7950]
### What version of Remix are you using?

2.1.0

### Are all your remix dependencies & dev-dependencies using the same version?

- [X] Yes

### Steps to Reproduce

Run `create-remix` with a version other than `@latest`: `npx create-remix@2.0.0`

### Expected Behavior

`create-remix` should use the specified version of Remix (and `create-remix`).

Not 100% sure but I think this is new behavior. I know the templates are maintained to fit the latest version of Remix, but I believe `create-remix` would still run in the specified version instead of always on the latest.

### Actual Behavior

`create-remix` uses the latest version instead of the specified version:

```bash
Need to install the following packages:
  create-remix@2.0.0
Ok to proceed? (y) y

 remix   v2.2.0 💿 Let's build a better website...

   dir   Where should we create your new project?
         ./my-remix-app
```

**Comment by brophdawg11** [comment]
ping @markdalgleish who did some work related to this recently.  I _think_ `create-remix` used to always instantiate your newly created app to the latest version.  I think Mark said he had made a change so it would instantiate the installed app to the version of `create-remix` being used - it's possible that only went out in 2.2.0 (or hasn't been released yet?) so might need to wait for 2.3.0 to be released before we can test it in the wild.

**Comment by markdalgleish** [comment]
This is fixed for v2.2.0 onwards. For prior versions you can pass the `-v` arg: `npx create-remix@latest -v 2.0.0`

**Comment by andrelandgraf** [comment]
Awesome, thanks @markdalgleish!
